evalrs/evaluation/utils.py

```diff
@@ -17,7 +17,7 @@
 from datetime import datetime
 
 
-LFM_DATASET_PATH="https://cikm-evalrs-dataset.s3.us-west-2.amazonaws.com/evalrs_dataset.zip"
+LFM_DATASET_PATH="https://evarl-2022-public-dataset.s3.us-east-1.amazonaws.com/evalrs_dataset.zip"
```

Comment:
this is good, let's make this update. And at a second look, can we follow-up with another PR right after to nuke the evalrs folder? The users of this library does not really need this, right? Why would a general jurity users inherits all this when they install the library (back then we didn't have a place to store for the competition, hence it's addition here but now that's over and not needed). 

My recommendation is, proceed with this PR as is to fix the dangling dataset path. Then follow-up with an immediate PR to remove the evalrs folder. We can move that folder under huggingface/skadio